### PR TITLE
Added run status tab to dashboard

### LINKED ISSUE #323: Add status of run indicator to dashboard
### Desired capability or behavior.

Add a status of run indication in the dashboard:

1. Add another tab that summarizes the run. Make it the initial tab the user sees when the dashboard is opened. 
2. Include info like date and time, status of the run, num procs, aviary input file or script, (code?)
3. Add green check mark, yellow exclamation point, or red X to the top of all tabs to indicate status of run

### Is your feature request related to a problem? Please describe.

_No response_

### Associated Bug Report

_No response_

### PR DESCRIPTION
**Pull request by hschilling**
### Summary

Added a run status tab to the dashboard to give the user a quick overview of the run.

Made it the initial tab on dashboard launch

### Related Issues

- Resolves #323 

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

**Review by Kenneth-T-Moore — APPROVED** [review]

File: aviary/interface/reports.py [hunk 1]
```diff
@@ -46,6 +51,59 @@ def register_custom_reports():
...
+def run_status(prob):
+    """
+    Creates a JSON file that containts high level overview of the run
+
+    Parameters
+    ----------
+    prob : AviaryProblem
+        The AviaryProblem used to generate this report
+    """
+    # Note: Due to a possible bug in OpenMDAO, we need to assign Problem as the
```
Comment (Kenneth-T-Moore) [inline aviary/interface/reports.py:72]: Note: I think Bret fixed this bug.

File: aviary/interface/reports.py [hunk 2]
```diff
@@ -46,6 +51,59 @@ def register_custom_reports():
...
+    status['Script'] = sys.argv[0]
+    status['Optimizer'] = prob.driver._get_name()
+    status['Number of driver iterations'] = prob.driver.result.iter_count
+    status['Number of model evals'] = prob.driver.result.model_evals
+    status['Number of deriv evals'] = prob.driver.result.deriv_evals
+    status['Wall clock run time'] = runtime_formatted
+    status['Exit status'] = prob.driver.result.exit_status
+    status['Report generation date and time'] = time_stamp
+
+    with open(report_file, 'w') as f:
```
Comment (Kenneth-T-Moore) [inline aviary/interface/reports.py:103]: I think you want to make sure to only do this on rank 0 if you are in MPI.

**Review by Kenneth-T-Moore — CHANGES_REQUESTED** [review]
Requested a fix for mpi.

File: aviary/interface/reports.py (hunk 2 above)
Comment (hschilling) [reply aviary/interface/reports.py:103]: Agreed! Fixed

**Comment by Kenneth-T-Moore** [comment]
Looks good. With Bret's fix, we don't need these anymore either:

```
    from aviary.interface.methods_for_level2 import AviaryProblem
    if not isinstance(prob, AviaryProblem):
        return
```

**Review by Kenneth-T-Moore — APPROVED** [review]

**Review by johnjasa — APPROVED** [review]

**Pull request merged**